spec/surfnperf_spec.js

```diff
@@ -1096,5 +1096,74 @@ define('spec/surfnperf_spec', [
 
     });
 
+
+    describe('#chromeLoadTimes', function() {
+      it('returns a reference to window.chrome.loadTimes() if the browser supports it, otherwise undefined', function() {
+        if(window.chrome && window.chrome.loadTimes()) {
```

Comment:
`if(window.chrome && window.chrome.loadTimes()) {` checks if the return of the `chrome.loadTimes()` function is truthy. What you want to do is check if `window.loadTimes` is truthy (i.e. it exists), so remove the `()`:
`if(window.chrome && window.chrome.loadTimes) {`